server/accounts.go

```diff
@@ -3990,20 +3991,23 @@ func (dr *DirAccResolver) Start(s *Server) error {
 	defer dr.Unlock()
 	dr.Server = s
 	dr.operator = opKeys
+	fetchTimeout := dr.fetchTimeout
 	dr.DirJWTStore.changed = func(pubKey string) {
 		if v, ok := s.accounts.Load(pubKey); ok {
 			if theJwt, err := dr.LoadAcc(pubKey); err != nil {
-				s.Errorf("update got error on load: %v", err)
+				s.Errorf("RESOLVER - Update got error on load: %v", err)
```

Comment:
Do we have cases where we all caps things similar? Maybe DirResolver?

Reply:
I used all caps so that it would look similar to `SYSTEM` logs, but I will update to `DirResolver` instead which would be clearer

Reply:
Updated to DirResolver to label these instead, logs now look like this: https://gist.github.com/wallyqs/9c968143014f2e5fb61b7d99ec0f5326